Foundation/NSNotificationQueue.swift

```diff
@@ -31,6 +33,16 @@ public class NSNotificationQueue : NSObject {
     internal let notificationCenter: NSNotificationCenter
     internal var asapList = NSNotificationList()
     internal var idleList = NSNotificationList()
+    internal lazy var idleRunloopObserver: CFRunLoopObserverRef = {
+        return CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CUnsignedLong(1) << 5, true, 0) {[weak self] observer, activity in
+            self!.notifyQueues(.PostWhenIdle)
+        }
+    }()
+    internal lazy var asapRunloopObserver: CFRunLoopObserverRef = {
+        return CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CUnsignedLong(1) << 1 | CUnsignedLong(1) << 7, true, 0) {[weak self] observer, activity in
```

Comment:
Thats because swift does not import the CF enums like it does on darwin. See https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/NSObjCRuntime.swift @ line 13 for an example of how to make it compatible.